Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 authors = ["Nicolas Patry <patry.nicolas@protonmail.com>"]
 name = "bindgen_cuda"
-version = "0.1.5"
+version = "0.1.7"
```

```suggestion
version = "0.1.5"
```
We can increment the version in a separate PR when creating a new release.